ui/src/components/LoadingIndicator.js

```diff
@@ -0,0 +1,18 @@
+import React, {Component} from 'react';
+import {FormattedMessage, defineMessages, injectIntl} from 'react-intl';
+
+const messages = defineMessages({
+  loading: {
+    id: 'LoadingIndicator.loading',
+    defaultMessage: 'Loading...',
+    description: 'A message indicating to the user that the page is loading.',
+  },
+});
+
+class LoadingIndicator extends Component {
+  render() {
+    return <FormattedMessage {...messages.loading} />
```

Comment:
Nit: Append a semicolon, I guess?

Reply:
I changed it to the just-a-function version (like the Footer and App components).